hack/ci/update-docs.sh

```diff
@@ -61,6 +61,26 @@ make runbook
 # update CRDs reference
 hack/render-crds.sh
 
+# update components page
+components_file=content/kubermatic/main/architecture/compatibility/KKP-components-versioning/_index.en.md
+cat > ${components_file} << EOT
++++
+title = "KKP Components"
+date = 2021-04-13T20:07:15+02:00
+weight = 2
+
++++
+
+## Kubermatic Kubernetes Platform Components
+
+The following list is only eligible for the version that is currently available. Kubermatic has a strong emphasis on the security and reliability of the provided software and releases updates regularly that also update components.
```

Comment:
maybe "applicable" is better than "eligible". Eligible sounds like "available to be elected in a vote".